lib/gds_api/support_api.rb

```diff
@@ -21,6 +21,26 @@ def create_global_export_request(request_details)
     post_json!("#{endpoint}/anonymous-feedback/global-export-requests", global_export_request: request_details)
   end
 
+  # Create a Page Improvement
+  #
+  # Makes a +POST+ request to the support api to create a Page Improvement.
+  #
+  # @param params [Hash] Any attributes that relate to a Page Improvement.
```

Comment:
It looks like the type can be before or after the arguments. The docs generate correctly. As the rest of the YARD annotations in this repo have the arguments first I favour keeping it this way. Are you looking for a change here?